specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchindex.json

```diff
@@ -0,0 +1,2720 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SearchIndexClient",
+    "description": "Client that can be used to query an index and upload, merge, or delete documents.",
+    "version": "2023-07-01-Preview",
```

Comment:
I think `preview` is usually lower-case?
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning-date-based-versioning